supervisor/shared/filesystem.c

```diff
@@ -69,6 +69,23 @@ static void make_empty_file(FATFS *fatfs, const char *path) {
     f_close(&fp);
 }
 
+
+static void make_sample_code_file(FATFS *fatfs) {
+    FIL fs;
+    UINT char_written = 0;
+    const byte buffer[] = "print('Hello World!')\n";
+
+    #if CIRCUITPY_FULL_BUILD == 0
+    make_empty_file(&vfs_fat->fatfs, "/code.py");
```

Comment:
`&vfs_fat->fatfs` should be just `fatfs`

Reply:
Thanks @jepler :)